src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -668,11 +652,7 @@
   "ThisWillApplyToAllIndexersPleaseFollowTheRulesSetForthByThem": "This will apply to all indexers, please follow the rules set forth by them",
   "Time": "Time",
   "TimeFormat": "Time Format",
-  "Timeleft": "Time Left",
   "Title": "Title",
-  "Titles": "Titles",
-  "Today": "Today",
-  "Tomorrow": "Tomorrow",
```

Comment:
Why deleting these? Are they not used on the calendar page?

Reply:
Per Nit's script that are not used.

per a search of GitHub they are not used.  "Today" is no where in the UI.

there is "'Today, " which is not translated

https://github.com/Readarr/Readarr/blob/6fa56b88733dd38687401304280a7e40377b58fe/frontend/src/Utilities/Date/formatDateTime.js